common/components/register_model/spec.yaml

```diff
@@ -52,12 +53,17 @@ inputs:
     optional: false
     description: Path to the model directory
 
```

Comment:
can we also add an optional model_version field please? If provided we would use that or will do auto version increment

Reply:
Added